### DISCUSSION TITLE
Refresh token and cookie on every authorised request

### DISCUSSION THREAD

**Discussion by stjefim** [fastapi-users/fastapi-users#1384] [Q&A]
Is it possible to refresh token and cookie on every (or at some) authorised request, i.e. if request is received and cookies/token are still valid then send Set-Cookie header to update cookie expiry time and update token expiry time in database.

Motivation: I believe default behaviour of many websites is to expire token not after some time, but after some inactivity.

**Comment by diegocgaona** [comment]
I would like to do this too, for example set expire for 14 days, if the users login on the 10 day, the cookie renews and get more 14 days.
It is possible?

**Comment by hgalytoby** [comment]
update cookies

https://github.com/fastapi-users/fastapi-users/discussions/1397
